Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'activesupport', '~> 5.0'
 gem 'activerecord',  '~> 5.0'
 
 # Fetch private dependencies from git
-gem 'acts_as_manual_list',          git: 'git@github.com:iknow/acts_as_manual_list'
-gem 'iknow_params',                 git: 'git@github.com:iknow/iknow_params'
-gem 'cerego_active_record_patches', git: 'git@github.com:iknow/cerego_active_record_patches'
-gem 'deep_preloader',               git: 'git@github.com:iknow/deep_preloader'
+gem 'acts_as_manual_list',          git: 'git@github.com:iknow/acts_as_manual_list',          branch: 'master'
```

Comment:
Motivation for adding explicit branches is missing. Consider seperate commit to explain.